pageserver/src/virtual_file.rs

```diff
@@ -975,7 +975,10 @@ impl VirtualFileInner {
     ) -> (FullSlice<B>, Result<usize, Error>) {
         let file_guard = match self.lock_file().await {
             Ok(file_guard) => file_guard,
-            Err(e) => return (buf, Err(e)),
+            Err(e) => {
+                println!("ERRORED :(");
```

Comment:
stray debugging code?

Reply:
please remove

also for future: `dbg!(e)`

Reply:
3d4052f253f9b1f6233d670753248fa8ef4bac41